sessionctx/variable/variable.go

```diff
@@ -305,10 +305,6 @@ func (sv *SysVar) validateScope(scope ScopeFlag) error {
 // may be less sophisticated in normalizing values. But errors should be caught and handled,
 // because otherwise there will be upgrade issues.
 func (sv *SysVar) ValidateWithRelaxedValidation(vars *SessionVars, value string, scope ScopeFlag) string {
-	warns := vars.StmtCtx.GetWarnings()
```

Comment:
Why it is removed?

Reply:
This is my fault, but StatementContext.SetWarnings should initialize errorCount instead of increasing,so i add `sc.mu.errorCount = 0`